### PR TITLE
gibbing  looks better now and Drops contained items on body parts

### PR DESCRIPTION
**Pull request by Bod9001**
pretty much says what it does on the tin, 

CL: [Improvement] gibbing has been visually improved. Organs now scatter from the body and unequip associated clothing.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.Move.cs [hunk 1]
```diff
@@ -366,7 +366,7 @@ public void Throw(ThrowInfo info)
 		//add player momentum
 		float playerMomentum = 0f;
 		//If throwing nearby, do so at 1/2 speed (looks clunky otherwise)
-		float speedMultiplier = Mathf.Clamp(correctedInfo.WorldTrajectory.magnitude / (throwRange <= 0 ? 1 : throwRange), 0.6f, 1f);
+		float speedMultiplier = Mathf.Clamp(correctedInfo.WorldTrajectory.magnitude / (throwRange <= 0 ? 1 : throwRange), 0.2f, 1f);
```
Comment (ThatDan123) [inline UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.Move.cs:369]: how is this change with normal throwing?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.Move.cs:369]: humm, only difference I noticed is that stuff like ID cards throw really slowly if you throw them close, but I just imagine that as them sliding along the floor for ages

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.Move.cs:369]: if I don't have this it snaps beyond the target position too fast, and instantly stops, make it look like it just appears there  , if the target position is close to the origin of the body

Comment (corp-0) [reply UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.Move.cs:369]: Should we make throw force a parameter?

Comment (pyrosshade) [reply UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.Move.cs:369]: > 
> 
> Should we make throw force a parameter?

A bit odd that we don't already.

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Core/Transform/CustomNetTransform.Move.cs:369]: I think one of the issues is that it expects a target position, to slow down in such not too sure though the code is a bit hard to read

**Review by NoooneyDude — APPROVED** [review]

**Review by corp-0 — APPROVED** [review]

**Pull request merged**